frontendUiImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/GitMacheteErrorHandler.java

```diff
@@ -25,9 +42,75 @@ public boolean submit(
-        ModalityState.NON_MODAL);
+    String reportBody = getReportBody(events, additionalInfo);
+
+    try {
+      val uri = constructNewGitHubIssueUri(reportBody);
+
+      GuiUtils.invokeLaterIfNeeded(() -> BrowserUtil.browse(uri), ModalityState.NON_MODAL);
+    } catch (URISyntaxException e) {
+      LOG.error("Cannot construct URI to open new bug issue! Erroneous report body: ${reportBody}", e);
+    }
     return true;
   }
+
+  private URI constructNewGitHubIssueUri(@Nullable String reportBody) throws URISyntaxException {
+    val uriBuilder = new URIBuilder("https://github.com/VirtusLab/git-machete-intellij-plugin/issues/new");
+    uriBuilder.addParameter("labels", "bug");
+    if (reportBody != null) {
+      uriBuilder.addParameter("body", reportBody);
+    } else {
+      uriBuilder.addParameter("template", "bug_report.md");
+    }
+    return uriBuilder.build();
+  }
+
+  private @Nullable String getReportBody(
+      IdeaLoggingEvent[] events,
+      @Nullable String additionalInfo) {
+    String reportBody = null;
+    try {
+      reportBody = getBugTemplate();
+      for (java.util.Map.Entry<String, String> entry : getTemplateVariables(events, additionalInfo).entrySet()) {
+        reportBody = reportBody.replace("%${entry.getKey()}%", entry.getValue());
+      }
+    } catch (IOException | AssertionError e) {
+      LOG.error("Error while creating bug report template", e);
+    }
+    return reportBody;
+  }
+
+  private String getBugTemplate() throws IOException, AssertionError {
+    val classLoader = GitMacheteErrorHandler.class.getClassLoader();
+    assert classLoader != null;
+    return IOUtils.resourceToString("bug_report.md", StandardCharsets.UTF_8, classLoader);
+  }
+
+  private java.util.Map<String, String> getTemplateVariables(
+      IdeaLoggingEvent[] events,
+      @Nullable String additionalInfo) {
+    val templateVariables = new java.util.HashMap<String, String>();
+    // Ide version, ie. Intellij Community 2021.3.1
+    templateVariables.put("ide", ApplicationInfo.getInstance().getFullApplicationName());
+    // Plugin version, ie. 1.1.1-10-SNAPSHOT git.c9a0e89-dirty
+    IdeaPluginDescriptor pluginDescriptor = PluginManagerCore.getPlugin(PluginId.getId("com.virtuslab.git-machete"));
+    if (pluginDescriptor != null) {
+      templateVariables.put("macheteVersion", pluginDescriptor.getVersion());
+    }
+    // OS name and version
+    val osName = SystemUtils.OS_NAME != null ? SystemUtils.OS_NAME : "";
+    val osVersion = SystemUtils.OS_VERSION != null ? SystemUtils.OS_VERSION : "";
+    templateVariables.put("os", "${osName} ${osVersion}");
+    // Additional info about error
+    templateVariables.put("additionalInfo", additionalInfo != null ? additionalInfo : "N/A");
+    // Error logs for events
+    val sep = System.lineSeparator();
+    String logs = Arrays.stream(events)
+        .map(IdeaLoggingEvent::getThrowableText)
```

Comment:
Would it make sense to fill up the `title` in the new-issue URL with joined `IdeaLoggingEvent#getMessage`s ?

Reply:
I will check how this looks like and if it will be somewhat descriptive I will change it.

Reply:
Yeah, if it is sth like exception class and message that would ideal for the issue title